calendarpal/src/test/java/org/dmfs/android/calendarpal/predicates/CalendarScopedPredicateTest.java

```diff
@@ -70,30 +73,13 @@ public void testArguments() throws Exception
         TransactionContext tContextDummy = dummy(TransactionContext.class);
         SoftRowReference<CalendarContract.Calendars> rowReferenceDummy = dummy(SoftRowReference.class);
         RowSnapshot<CalendarContract.Calendars> calendarRow = failingMock(RowSnapshot.class);
-        Predicate predicate = failingMock(Predicate.class);
-        Predicate.Argument argument = failingMock(Predicate.Argument.class);
-
         doReturn(rowReferenceDummy).when(calendarRow).reference();
-
-        doReturn(new SingletonIterable<>(argument)).when(predicate).arguments(tContextDummy);
-
-        doReturn(new Present<>(3)).when(argument).backReference();
-        doReturn("5").when(argument).value();
-
         doReturn(new BackReference<>(dummy(Uri.class), 12)).when(tContextDummy).resolved(rowReferenceDummy);
```

Comment:
I know that I added this `new BackReference(..)` usage here, but do you think we can replace it with something simpler? To have less class involved in the test.

Reply:
I've though about that too. I guess what we would need is a `PredicateMockReference` with returns a specific `Predicate` and a `ReferenceMockTransactionContext` which always returns the same `RowReference`.

Reply:
Ok, I suppose we can get back to this when needed for another test, or maybe in the coverage increasing tasks, so that we can merge this PR.